orm/index_test.go

```diff
@@ -362,14 +369,14 @@ func TestNullableIndex(t *testing.T) {
 			db := store.MemStore()
 			for _, init := range tc.setup {
 				err := uniq.Update(db, nil, init)
-				require.NoError(t, err)
+				assert.Nil(t, err)
 			}
 
 			err := uniq.Update(db, tc.prev, tc.next)
 			if tc.isError {
-				require.Error(t, err)
+				assert.Equal(t, true, err != nil)
```

Comment:
I guess we'll need to convert the `tc.isError` here to actual error and use `assert.IsErr`